### PR TITLE
Add tooltips for some browser columns

### PR DESCRIPTION
**Pull request by RumovZ**
My main motivation was to solve the confusion about the "Card/Note Modified" columns, but some other clarifications seemed to be advisable as well.

### CODE REVIEW

File: ftl/core/browsing.ftl [hunk 1]
```diff
@@ -149,6 +149,13 @@ browsing-sidebar-save-current-search = Save Current Search
 browsing-sidebar-card-state = Card State
 browsing-sidebar-flags = Flags
 browsing-today = Today
+browsing-tooltip-card-modified = The last time changes were made to a card, including reviews, flags and deck changes
+browsing-tooltip-note-modified = The last time changes were made to a note, usually field content or tag edits
+browsing-tooltip-card = The name of a card's card template
+browsing-tooltip-cards = The number of cards to a note
```
Comment (dae) [inline ftl/core/browsing.ftl:155]:
```suggestion
browsing-tooltip-cards = The number of cards this note has
```

File: ftl/core/browsing.ftl [hunk 2]
```diff
@@ -149,6 +149,13 @@ browsing-sidebar-save-current-search = Save Current Search
...
+browsing-tooltip-notetype = The name of a note's notetype
```
Comment (dae) [inline ftl/core/browsing.ftl:156]:
```suggestion
browsing-tooltip-notetype = The type of this note
```

File: ftl/core/browsing.ftl [hunk 3]
```diff
@@ -149,6 +149,13 @@ browsing-sidebar-save-current-search = Save Current Search
 browsing-sidebar-card-state = Card State
 browsing-sidebar-flags = Flags
 browsing-today = Today
+browsing-tooltip-card-modified = The last time changes were made to a card, including reviews, flags and deck changes
+browsing-tooltip-note-modified = The last time changes were made to a note, usually field content or tag edits
```
Comment (dae) [inline ftl/core/browsing.ftl:153]:
```suggestion
browsing-tooltip-note-modified = The last time changes were made to this note, usually field content or tag edits
```

File: ftl/core/browsing.ftl [hunk 4]
```diff
@@ -149,6 +149,13 @@ browsing-sidebar-save-current-search = Save Current Search
 browsing-sidebar-card-state = Card State
 browsing-sidebar-flags = Flags
 browsing-today = Today
+browsing-tooltip-card-modified = The last time changes were made to a card, including reviews, flags and deck changes
```
Comment (dae) [inline ftl/core/browsing.ftl:152]:
```suggestion
browsing-tooltip-card-modified = The last time changes were made to a/the card, including reviews, flags and deck changes
```

Comment (dae) [reply ftl/core/browsing.ftl:152]: This reads a bit awkwardly - in the notes mode case it should presumably be "a card", but in the cards mode case "this card". Better suggestions welcome!

**Review by dae — COMMENTED** [review]
Code changes look good Rumo, just some bikeshedding on the wording

**Comment by RumovZ** [comment]
You see these tooltips in the table's header context menu, so it sounds strange to me to refer to "***this/the*** card/note". And as you've pointed out, it doesn't always work for notes mode.
Maybe we can circumvent these pitfalls with "... card changes were made ...", "The notetype", "The number of cards", etc.
In the end, you're the native speaker, so let me know what works best for you.

**Comment by dae** [comment]
Sorry, you're quite right. I saw in the code they're column headers, but then for some reason I kept picturing the tooltips as appearing over individual rows. 😅

**Pull request merged**